tests/dummy/app/router.js

```diff
@@ -1,7 +1,9 @@
 import Ember from 'ember'
+const {Router: EmberRouter} = Ember
+
 import config from './config/environment'
 
-const Router = Ember.Router.extend({
+const Router = EmberRouter.extend({
```

Comment:
Was this done with an old version of the eslint plugins? I thought we weren't doing this anymore, or is it only for `Object` and `String`? I'd personally rename the `const Router = ` to something like `const MyRouter = ` and leave `Router` for the thing destructured, but I guess it's not that big a deal.